package/scripts/preinstall

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Checked for installed CLT
+if [[ -e "/Library/Developer/CommandLineTools/usr/bin/git" ]]; then
+  exit 0
+fi
+
+if [[ -z "$COMMAND_LINE_INSTALL" ]]; then
+  printf "No auto-install of CommandLine Tools in headless install!"
```

Comment:
There is a hack you can do to do this automatically: https://github.com/MikeMcQuaid/strap/blob/ff0fb263a3ac390fa85a3aa0a757df44a87bdd6a/bin/strap.sh#L244-L269

Similarly think there's one in the official Homebrew installer.